app/assets/javascripts/snowcrash/modules/issues/table.js.coffee

```diff
@@ -108,17 +114,36 @@ class IssueTable
         data: data,
         dataType: 'json',
         success: (data) ->
+          $that.prop('checked', false)
           issue_id = $that.val()
 
           $($row.find('td')[that.tagColumnIndex]).replaceWith(data.tag_cell)
           $("#issues #issue_#{issue_id}").replaceWith(data.issue_link)
-          if $('input[type=checkbox]:checked').length == 0
-            $('.js-issue-actions').css('display', 'none')
+          if $('input[type=checkbox]:checked.js-multicheck').length == 0
+            that.resetToolbar()
 
         error: (foo,bar,foobar) ->
           $($row.find('td')[that.tagColumnIndex]).replaceWith("<td class='text-error'>Please try again</td>")
       }
 
+  onMergeSelected: (event) ->
```

Comment:
You are right. I spent some time on this.
I decided to use a POST to avoid super long urls when we are merging many issues. I thought about being able to merge a few dozens of issues...
@etdsoft what's your opinion on this?